functions/completed-checks/v1.js

```diff
@@ -93,7 +93,7 @@ async function handleCompletedCheck (context, completedCheckMessage) {
     PartitionKey: completedCheckMessage.checkCode,
     RowKey: uuid(),
     eventType: 'completed-check',
-    payload: JSON.stringify(completedCheckMessage),
+    payload: '<removed>',
```

Comment:
the conversation in the support meeting makes me consider doing this, as in a replay situation it could be useful.

Reply:
Now added